src/base.js

```diff
@@ -1,30 +1,29 @@
-import assign from 'lodash.assign'
-import diff from 'immutablediff'
-import forEach from 'lodash.foreach'
-import isArray from 'lodash.isarray'
-import keys from 'lodash.keys'
-import wrap from 'lodash.wrap'
 import defer from 'lodash.defer'
```

Comment:
Yeah it's basically just a `setTimeout(cb, 1)` call https://github.com/lodash/lodash/blob/master/defer.js